script_editor/app/views/plays/show.html.erb

```diff
@@ -125,6 +125,51 @@
             <!-- This button shows the play's analytics -->
             <button class="nav-analytics" id="analytics-button">ANALYTICS</button>
 
+            <!-- This button toggles the strikethrough edits' visibility -->
+            <button class = "nav-synopsis" curState = "showing" id="viewToggle-button"> Hide Edits </button>
+
+            <script>
+                var toggleBtn = document.getElementById("viewToggle-button");
+                toggleBtn.curState = "showing";
+                toggleBtn.onclick = function() {
+                    var words = Array.from(document.getElementsByClassName("word"));
+                    var punctuation = Array.from(document.getElementsByClassName("punc"));
+                    var directions = Array.from(document.getElementsByClassName("stage"));
+                    var cuttableScript = words.concat(punctuation).concat(directions);
+                    cuttableScript.forEach(function(word) {
+                        if (toggleBtn.curState == "showing") // User wishes to hide edit
+                        {
+                            if (word.dataset.cut == "true")
```

Comment:
It seems to me that these nested if statements could be handled by `if (a && b) else` instead of `if (a) { if (b) } else`